examples/exclude-words/create_wordlist.py

```diff
@@ -0,0 +1,54 @@
+import argparse
+import glob
+import json
+import pathlib
+from pdfminer.high_level import extract_text
+
+
+def main(args):
+    file_path = args.file_path
+    frequency = int(args.minimal_frequency)
+    minimal_length = int(args.minimal_length)
+    words_file = args.words_file
+
+    if not file_path[-1] == '/':
+        file_path += '/'
```

Comment:
We can use pathlib and check if the path is a directory instead: `Path(...).is_dir()` Specify the Path type in `parser.add_argument`